worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/listener/ChunkListener.java

```diff
@@ -60,9 +60,6 @@ public abstract class ChunkListener implements Listener {
             Settings.settings().TICK_LIMITER.FALLING, Settings.settings().TICK_LIMITER.ITEMS};
 
     public ChunkListener() {
-        if (FoliaSupport.isFolia()) {
```

Comment:
That feature doesn't make sense on Folia, it should always be disabled

Reply:
This event produce some issues on folia. The normal way before is not longer working here